src/stdune/path.mli

```diff
@@ -1,30 +1,24 @@
-(** Relative path with unspecified root *)
+(** Relative path without restrictions *)
```

Comment:
Propose clarification:

```
(** Relative path with unspecified base.
Both "." and ".." are allowed and no normalization is performed. *)
```

By the way currently trailing "/" and repeated "/" are also allowed.